README.md

```diff
@@ -14,148 +14,80 @@ Metro Board Reports is a member of the [Councilmatic family](https://www.council
 
-* Python 3.4
-* PostgreSQL 9.4 +
+### Install OS level dependencies:
 
-**Install app requirements**
+* [Docker](https://www.docker.com/get-started)
 
-We recommend using [virtualenv](http://virtualenv.readthedocs.org/en/latest/virtualenv.html) and [virtualenvwrapper](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) for working in a virtualized development environment. [Read how to set up virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
-
-Once you have virtualenvwrapper set up, run the following in your terminal:
-
-```bash
-mkvirtualenv la-metro
-git clone git@github.com:datamade/la-metro-councilmatic.git
-cd la-metro-councilmatic
-pip install -r requirements.txt
-```
-
-Afterwards, whenever you want to use this virtual environment, run `workon la-metro`.
-
-**Special installation requirements**
-
-Metro runs a custom script that converts board-report attachments into plaintext. The script uses `textract` for conversions. Mac users can [follow the instructions given by `textract`](http://textract.readthedocs.io/en/stable/installation.html#osx).
-
-For Ubuntu users and on the server, the `textract` installation instructions require a little remediation. Follow these steps:
-
-(1) [Install all the dependencies](http://textract.readthedocs.io/en/stable/installation.html#ubuntu-debian).
-
-(2) `textract` fails when installing `pocketsphinx`. Happily, `textract` does not need this dependency for converting PDFs or word documents into plain text. [This issue offers a script that installs a "fake pocketsphinx"](https://github.com/deanmalmgren/textract/pull/178). Do as the issue instructs.
-
-(3) You're ready! Inside your virtualenv execute: `pip install textract==1.6.1`.
-
-
-**Create your settings file**
+### Run the application
 
 ```bash
-cp councilmatic/settings_deployment.py.example councilmatic/settings_deployment.py
+docker-compose up -d
 ```
 
-Then edit `councilmatic/settings_deployment.py`:
-- `USER` should be your username
+### Load in the data
 
-**Setup your database**
-
-Before we can run the website, we need to create a database.
+Every hour, DataMade scrapes the Legistar Web API and makes the results available on the Open Civic Data API, which hosts standardized data patterns about government organizations, people, legislation, and events. Metro Board Reports relies upon this data.
 
-```bash
-createdb lametro
-```
+The django-councilmatic app comes with an `import_data` management command, which populates your database with content loaded from the OCD API. You can explore the nitty-gritty of this code [here](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/management/commands/import_data.py).
 
-Then, run migrations:
+Run the `import_data` command, which may take a few minutes to an hour, depending on how much data you need to import:
```

Comment:
Adding in a link to the options (such as `--update-since`) would be good here, or a note directing people to the Wiki.